### DISCUSSION TITLE
Creality Ender-3 BL Touch

### DISCUSSION THREAD

**Discussion by Madajprod** [supermerill/SuperSlicer#943] [Q&A]
![CaptureEnderBLTouch](https://user-images.githubusercontent.com/25423021/106760847-832c8e80-6634-11eb-828c-3dc93b667b6f.JPG)
Hello,
I wanted to test the new profiles added for the Ender-3, including the "Ender-3 BLTouch" series
But due to requesting the addition of these profiles, impossible to finalize, because it cannot find an associated filament profile ...
I can click on add with the default filaments of FFF printers, impossible to finalize the configuration if we add the BLTouch profiles (while the classic ones work)

**Comment by supermerill** [comment]
should work. open an issue if you have a bug like that.
